intro/07_symbols/sites.rb

```diff
@@ -7,4 +7,5 @@
 ]
 
 # Your code goes here
-
+url_map = Hash.new
+urls.each { |url| url_map[url.to_sym] = open(url) }
```

Comment:
Close, but slightly different. `open(url)` will give you the response of the webpage but not its size. You to add a little more here.